README.md

```diff
@@ -41,7 +41,7 @@ The primary source of information. Retrieves:
 
 Optionally used if creds are provided. Retrieves:
 
-* [Whois](https://api.riskiq.net/api/whois_pt/)
+* [Whois](https://api.riskiq.net/api/whois_pt/) [(registration)](https://community.riskiq.com/registration/wtfis)
```

Comment:
Seems to work as well with just a trailing slash: https://community.riskiq.com/registration/ . Mind updating to this? Thanks for the PR!